### PR TITLE
Update changelog for v8.1.5

### PR DESCRIPTION
**Pull request by danjm**
Updates the changelog for v8.1.5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,14 @@
...
 ## 8.1.5 Wed Nov 18 2020
-- [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
+- [#9871](https://github.com/MetaMask/metamask-extension/pull/9871): Show send text upon hover in main asset list
+- [#9855](https://github.com/MetaMask/metamask-extension/pull/9855): Make edit icon and account name in account details modal focusable
+- [#9853](https://github.com/MetaMask/metamask-extension/pull/9853): Provide alternative text for images where appropriate
+- [#9869](https://github.com/MetaMask/metamask-extension/pull/9869): Remove CoinSwitch from the Deposit modal
+- [#9883](https://github.com/MetaMask/metamask-extension/pull/9883): Move add contact button in fullscreen/expanded view of settings lower to expose the close button.
+- [#9891](https://github.com/MetaMask/metamask-extension/pull/9891): Add token verification message to swaps build quote screen
+- [#9896](https://github.com/MetaMask/metamask-extension/pull/9896): Show failed token balance updates
+- [#9909](https://github.com/MetaMask/metamask-extension/pull/9909): Update asset page etherscan link to the address-filtered token page on Etherscan
```
Comment (Gudahtt) [inline CHANGELOG.md:13]: #9910 is also user facing, so we should probably include it as well:
```suggestion
- [#9909](https://github.com/MetaMask/metamask-extension/pull/9909): Update asset page etherscan link to the address-filtered token page on Etherscan
- [#9910](https://github.com/MetaMask/metamask-extension/pull/9910): Revert "Show a 'send eth' button on home screen in full screen mode"
```

**Review by Gudahtt — APPROVED** [review]
Nice descriptions! I could only find the one user-facing change that was missing.

File: CHANGELOG.md (hunk 1 above)
Comment (danjm) [reply CHANGELOG.md:13]: #9910 never made it to prod though

Comment (danjm) [reply CHANGELOG.md:13]: I mean, the PR it is reverting never made it to prod

Comment (Gudahtt) [reply CHANGELOG.md:13]: It was shipped in v8.1.4: https://github.com/MetaMask/metamask-extension/commit/64657efdefd98451ba2d5b56cc45d20142808ff4

It's also in the changelog for v8.1.4

Comment (danjm) [reply CHANGELOG.md:13]: oh, missed that somehow

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**